v1/definition.yaml

```diff
@@ -0,0 +1,172 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: MediaWiki Definition API
+  description: Wikitionary word definition API
+  termsofservice: https://github.com/wikimedia/restbase#restbase
+  contact:
+    name: Services
+    email: services@lists.wikimedia.org
+    url: https://www.mediawiki.org/wiki/Services
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /definition/{term}:
+    get:
+      tags:
+        - Page content
+      description: >
+        Returns the definition of the term based on the latest wikitionary page content
+        available in storage.
+
+        Provide a `Cache-Control: no-cache` header to request the latest data.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json
+      parameters:
+        - name: term
+          in: path
+          description: The term to define
+          type: string
+          required: true
+      responses:
+        '200':
+          description: The definition for the given term
+          schema:
+            $ref: '#/definitions/definitionsResponse'
+        '404':
+          description: Unknown term or definitions not supported for this language
+          schema:
+            $ref: '#/definitions/problem'
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+
+      x-setup-handler:
+        # Set up a simple key-value bucket.
+        - init:
+            method: 'put'
+            uri: /{domain}/sys/key_value/definition
+            body:
+              valueType: 'json'
+
+      x-request-handler:
+        # Get the revision metadata for access control (will return an error
+        # if there's no access), and so that we can use the normalized title &
+        # other info in case we have a storage miss.
+        - rev_info:
+            request:
+              method: get
+              uri: /{domain}/sys/page_revisions/page/{term}
+            response:
+              body: '{{rev_info.body.items[0]}}'
+
+          storage:
```

Comment:
So this will return the definition if it is found in storage. However, an older version might be stored here, which can create inconsistent results. I don't feel too strongly about this though, given that we have the hack in place to refresh this endpoint.